node/coinstacks/ethereum/api/src/app.ts

```diff
@@ -1,13 +1,20 @@
 import express, { json, urlencoded } from 'express'
 import cors from 'cors'
+import { ethers } from 'ethers'
 import { join } from 'path'
 import { Server } from 'ws'
 import swaggerUi from 'swagger-ui-express'
-import { middleware, ConnectionHandler } from '@shapeshiftoss/common-api'
+import { middleware, ConnectionHandler, Registry } from '@shapeshiftoss/common-api'
+import { Tx as BlockbookTx, WebsocketClient, getAddresses, NewBlock } from '@shapeshiftoss/blockbook'
 import { logger } from './logger'
 import { RegisterRoutes } from './routes'
+import { EthereumTx } from './models'
+import { handleBlock, handleTransactionWithInternalTrace } from './handlers'
 
-const port = process.env.PORT ?? 3000
+const PORT = process.env.PORT ?? 3000
```

Comment:
do we need this 3000 default?

Reply:
we actually never pass in PORT so I can just hardcode to 3000, just provides the flexibility if we ever need I guess